stubs/api/config/sanctum.php

```diff
@@ -17,7 +17,8 @@
         '%s%s%s',
         'localhost,localhost:3000,127.0.0.1,127.0.0.1:8000,::1',
         env('APP_URL') ? ','.parse_url(env('APP_URL'), PHP_URL_HOST) : '',
-        env('FRONTEND_URL') ? ','.parse_url(env('FRONTEND_URL'), PHP_URL_HOST) : ''
+        env('FRONTEND_URL') ? ','.parse_url(env('FRONTEND_URL'), PHP_URL_HOST) : '',
+        \Laravel\Sanctum\Sanctum::currentApplicationUrlWithPort()
```

Comment:
Can you import the class here like in https://github.com/laravel/laravel/blob/9.x/config/sanctum.php ?

Reply:
Yeah, just did it.